package.json

```diff
@@ -19,7 +19,7 @@
     "test:integration": "lerna run --stream test:integration",
     "bump-version": "lerna version --no-git-tag-version --no-push",
     "create-release-pr": "node packages/release-tool",
-    "postinstall": "lerna bootstrap"
+    "postinstall": "lerna bootstrap --ignore open-lens && lerna bootstrap --scope open-lens --include-dependencies"
```

Comment:
`open-lens` package needs to be bootstrapped as last because during install it creates `bin` wrappers from other packages in the monorepo. Without this hack bin wrappers are invalid on Windows on first install.